### ISSUE TITLE
expose pql API

### ISSUE THREAD

**Issue by faxm0dem** [voxpupuli/pypuppetdb#167]
I'd like to use the pql API, but it's currently protected _

**Comment by bastelfreak** [comment]
Hi @faxm0dem. Can you provide a PR for this to make it configureable?

**Comment by faxm0dem** [comment]
what do you mean configurable? either it's public, or not
I can make a PR for exposing the pql so we can use:

```python
puppetdb.pql(query='resources[certname] { type = "Class" and title = "Foo" }')
```

**Comment by faxm0dem** [comment]
Hi, I'm still not clear on what you're asking

**Comment by gdubicki** [comment]
Hey @faxm0dem ! We are open to the PR that will expose this API. :)

**Comment by gdubicki** [comment]
I created a draft PR with PQL API here: #201

**Comment by gdubicki** [comment]
Released in v2.4.0.rc1. Please feel free to test it, @faxm0dem !

**Comment by gdubicki** [comment]
Done with the release of final v2.4.0.
